Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/properties.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2017 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+// You can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// Licensees holding a valid commercial license may use this file in
+// accordance with the commercial license agreement provided with the software.
+//
+// This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
+// WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+//
+// $URL$
+// $Id$
+//
+//
+// Author(s)     : Andreas Fabri
+
+#ifndef CGAL_POLYGON_MESH_PROCESSING_PROPERTIES_H
+#define CGAL_POLYGON_MESH_PROCESSING_PROPERTIES_H
+
+#include <CGAL/license/Polygon_mesh_processing.h>
+
+namespace CGAL
+{
+enum face_patch_id_t            { face_patch_id };
+enum vertex_num_feature_edges_t { vertex_num_feature_edges };
+enum halfedge_is_feature_t      { halfedge_is_feature };
+enum vertex_selection_t         { vertex_selection};
+enum edge_selection_t           { edge_selection};
+enum face_selection_t           { face_selection};
```

Comment:
I see that that point was raised during the review.

https://cgal.geometryfactory.com/CGAL/Members/wiki/Features/BGL/1st_round

But the naming in existing packages of CGAL, at the time the "FaceGraph" was added, was not consistent.

Anyway:
- in a 2D planar graph (such as a 2D triangulation), that is common to call "face" the 2-faces,
- for a polyhedron, the correct term for 2-face is "facet", and "face" is for any dimension: vertices are 0-faces, edges are 1-faces, and so on...
- more generally, facets are faces of co-dimension 1: that means that their dimension is the ambient dimension minus 1,
- the naming for the BGL extension "FaceGraph" is ambiguous, because that concept does not imply any ambient dimension.

Let's forget that, and keep face, at it was decided during the review. At least that is consistent with the usual wording of a planar graph.